calico/openstack/test/__init__.py

```diff
@@ -0,0 +1 @@
+from calico.test import lib
```

Comment:
This file could do with a copyright statement, to match `calico/felix/test/__init__.py`.